string bag_profile_name
bool set
---
bool success
